src/Control/Alt.purs

```diff
@@ -2,5 +2,16 @@ module Control.Alt where
 
 infixl 3 <|>
 
+-- | The `Alt` type class identifies an associative operation on a type.
+-- | It is similar to `Semigroup`, except that it applies to types of
+-- | kind (* -> *), like Array or List, rather than concrete types like
+-- | String or Number.
```

Comment:
Sorry, really picky, but could you add backticks around the kind and `Array`, `List`, and so on types please?